service/component-samples/rv/src/main/java/org/fido/iot/sample/RvAppConstants.java

```diff
@@ -0,0 +1,23 @@
+package org.fido.iot.sample;
+
+import java.nio.file.Path;
+
+public class RvAppConstants {
+
+  public static final String RV_PORT = "8040";
+  public static final String DB_PORT = "8050";
+  public static final String DB_HOST = "localhost";
+  public static final String DB_USER = "sa";
+  public static final String DB_PASSWORD = "";
+
+  public static final String WEB_PATH = "/fido/100/msg";
+  public static final String DB_PATH =
+      Path.of(System.getProperty("user.dir"), "target", "data", "rvs").toString();
+  public static final String SERVER_PATH =
+      Path.of(System.getProperty("user.dir"), "target", "tomcat").toString();
+
+  public static final String CATALINA_HOME = "catalina.home";
+  public static final String DB_USER_KEY = "db.user";
```

Comment:
Please update the properties separator to '-', '_', from '.'. Dot separators are problematic while setting properties as environment variables.

Reply:
Done